### ISSUE TITLE
Appropriate loader handler for specific file only.

### ISSUE THREAD

**Issue by rohitp-bsf** [webpack-contrib/style-loader#513]
I have configured the style-loader for lazyloading of SCSS files are it works superbly. 

It causing the issue like `You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.` though the file name is same for all modules but showing an issue for 2 files only.

- Operating System: Any
- Node Version: v14.8
- NPM Version:
- webpack Version:
- style-loader Version:

### Expected Behavior

It should not show the error if the same naming files are loading properly.

### Code

```
	rules: [
			...wp_rules,
			{
				test: /\.(scss|css)$/,
				exclude: [/node_modules/, /style/],
				use: [
					{
						loader: 'style-loader',
						options: {
							injectType: 'lazySingletonStyleTag',
							attributes: { id: 'test-editor-styles' }
						},
					},
					'css-loader',
					'sass-loader',
				],
			},
		],
```

**Comment by alexander-akait** [comment]
Not reproducible, please provide all steps

**Comment by alexander-akait** [comment]
Closing due to inactivity. Please test with latest version. Thanks!

Also you don't provide version of `style-loader`, so I can't say what is wrong, anyway feel free to feedback and I will try to help you.
